scripts/utils.mjs

```diff
@@ -1,7 +1,9 @@
+import { MODULE_ID } from "../module.mjs";
 export {
   wait,
   updateTokenSight,
-  shuffleArray
+  shuffleArray,
+  sortArray
 }
```

Comment:
This looks good for now but we'll probably eventually need to make a dummy file at some point in the future that collects all these util functions into one for a single export.

Reply:
i agree that this can grow out of hand.